### PR TITLE
Update 1.create-native-index.md

### PR DESCRIPTION
**Pull request by whitewum**

### CODE REVIEW

File: docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md [hunk 1]
```diff
@@ -58,7 +68,7 @@ CREATE {TAG | EDGE} INDEX [IF NOT EXISTS] <index_name> ON {<tag_name> | <edge_na
 
   - 为**定长**字符串属性创建索引时，使用`prop_name`设置要索引的属性即可，索引长度为字符串长度。
   
-  - 为标签或边类型本身创建索引时，请忽略`prop_name_list`。
+  - 为标签或边类型本身创建索引时，忽略`prop_name_list`。
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md:71]: Great!

File: docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
+`CREATE INDEX`语句用于对Tag、EdgeType或其属性创建`原生索引`。通常分别称为`标签索引`、`边类型索引`、`属性索引`。
+
+!!! note
 
-大多数图查询都是从属性标识的点或边的列表开始检索的。索引可以让大规模图数据库的全局检索更加高效。
+    `标签索引`和`边类型索引`用于`和标签、边类型（自身）相关`的查询,而不是`基于标签上某属性`的查询。例如用 LOOKUP 找到`某种边类型E`的所有的边。
+    
+    `属性索引`目的是`基于属性的查询`。例如基于属性`Age`找到`Age == 19`的所有的点VID。
+    
+     如果已经为`某个标签T的属性A`建立过属性索引 `i_TA`，则没有必要对于`标签T`额外再建立一个标签索引`i_T`。这是因为查询引擎可以使用`i_TA`来替代`i_T`。边类型索引同理。但是，`i_T`却不能替代`i_TA`用于属性查找。
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md:15]: No need to use note here. These lines are normal instructions.

File: docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md [hunk 3]
```diff
@@ -72,15 +82,15 @@ nebula> CREATE TAG INDEX player_index on player();
...
 
-## 创建单个属性索引
+## 创建单属性索引
 
 ```ngql
 nebula> CREATE TAG INDEX player_index_0 on player(name(10));
 ```
 
-上述示例是为所有包含标签`player`的点创建属性`name`的索引，索引长度为10，即使用属性`name`的前10个字符创建索引。
+上述示例是为所有包含标签`player`的点创建属性`name`的索引，索引长度为10，即属性`name`的前10个字符来创建索引。
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md:93]: 删除使用后读不通顺啦

**Review by randomJoe211 — APPROVED** [review]

File: docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md (hunk 2 above)
Comment (whitewum) [reply docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md:15]: ok

File: docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md (hunk 3 above)
Comment (whitewum) [reply docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md:93]: ok

**Review by cooper-lzy — DISMISSED** [review]

File: docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md [hunk 4]
```diff
@@ -1,10 +1,18 @@
...
 
-大多数图查询都是从属性标识的点或边的列表开始检索的。索引可以让大规模图数据库的全局检索更加高效。
+    `标签索引`和`边类型索引`用于`和标签、边类型（自身）相关`的查询,而不是`基于标签上某属性`的查询。例如用 LOOKUP 找到`某种边类型E`的所有的边。
+    
+    `属性索引`目的是`基于属性的查询`。例如基于属性`Age`找到`Age == 19`的所有的点VID。
+    
+    如果已经为`某个标签T的属性A`建立过属性索引 `i_TA`，则没有必要对于`标签T`额外再建立一个标签索引`i_T`。这是因为查询引擎可以使用`i_TA`来替代`i_T`。边类型索引同理。但是，`i_T`却不能替代`i_TA`用于属性查找。
 
-!!! note "如何创建全文索引，请参见[部署全文索引](../../4.deployment-and-installation/6.deploy-text-based-index/2.deploy-es.md)"
+    如何创建全文索引，请参见[部署全文索引](../../4.deployment-and-installation/6.deploy-text-based-index/2.deploy-es.md)。
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md:15]: Why indent this section?

File: docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md [hunk 5]
```diff
@@ -1,10 +1,18 @@
+!!! warning 
+
+    索引的概念和使用限制都较为复杂。索引配合 `LOOKUP` 和 `MATCH` 语句使用。
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/14.native-index-statements/1.create-native-index.md:3]: Plz don't add any content above the H1.

**Review by randomJoe211 — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Review by FengUyu — APPROVED** [review]

**Pull request merged**